app/src/main/java/de/stephanlindauer/criticalmaps/Main.java

```diff
@@ -35,9 +35,12 @@
 
 public class Main extends AppCompatActivity implements NavigationView.OnNavigationItemSelectedListener {
 
+    private final static String KEY_NAVID = "main_navid";
```

Comment:
You might want to prefix all keys with the package name which is available from the `BuildConfig.APPLICATION_ID + "main_navid`.